daeun/lv1/두-정수-사이의-합.js

```diff
@@ -0,0 +1,21 @@
+function solution(a, b){
+    return (a+b)*(Math.abs(b-a)+1)/2; // 가우스 공식
+}
```

Comment:
와!!! 저도 어떤 방법으로 풀어야 좀 더 간단하게 풀 수 있을까 고민을 해봤는데 
가우스 공식은 진짜 생각도 못해봤네요!!
```suggestion
function solution(a, b){
    return (a+b)*(Math.abs(b-a)+1)/2; // 가우스 공식
}
```
좋은 코드 공유 감사합니다!! 👍

Reply:
오.. 가우스 공식 방법이 있었군요 저도 생각 치 못해봤습니다. 코드 잘보았습니다!

Reply:
@yjlee9909 @Han-Sang 저도 여러 풀이 방법을 찾아보다, 가우스 공식을 적용해 봤습니다. 비슷한 문제에서 자주 쓰일 수 있는 공식이라, 코드를 외워버리시면 도움이 많이 될 것 같습니다! 😊